main.go

```diff
@@ -108,7 +109,10 @@ func main() {
 	}
 
 	if versionFlag {
-		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH)
+		cmd := exec.Command("git", "--version")
+		stdout, _ := cmd.Output()
+		git_version := stdout[12:]
```

Comment:
This seems really fragile, I'd rather we split this by `' '` or something.

Also, what if the user somehow doesn't already have `git`, would this crash?